moveit_setup_assistant/moveit_setup_app_plugins/src/perception_config.cpp

```diff
@@ -57,7 +57,7 @@ void PerceptionConfig::loadPrevious(const std::filesystem::path& package_path, c
   }
   catch (const std::runtime_error& e)
   {
-    RCLCPP_ERROR_STREAM((*logger_), e.what());
+    RCLCPP_ERROR_STREAM(*logger_, e.what());
```

Comment:
Why is this a logger pointer?

Reply:
[The namespace for the logger is determined dynamically during the initialization phase](https://github.com/ros-planning/moveit2/blob/ab84da998e135fcb89e56f0ad1fbbb57a493c57c/moveit_setup_assistant/moveit_setup_framework/include/moveit_setup_framework/config.hpp#L72) and because it is loaded in via `pluginlib`, the constructor can't have any parameters.